pkg/api/secretbootstrap/secretboostrap.go

```diff
@@ -91,6 +102,64 @@ func (c *Config) UnmarshalJSON(d []byte) error {
 	return c.resolve()
 }
 
+func (c *Config) MarshalJSON() ([]byte, error) {
+	target := &configWithoutUnmarshaler{
+		VaultDPTPPrefix:           c.VaultDPTPPrefix,
+		ClusterGroups:             c.ClusterGroups,
```

Comment:
Technically this is being copied by reference, so this marshal will change the source `Config` - not sure if we care?

Reply:
I don't think we care currently. But, it would probably be better if it didn't do that...